domains/nooz.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal portfolio for nooz",
+    "repo": "https://github.com/NoozSBC/portfolio/",
+    "owner": {
+        "username": "NoozSBC",
+        "email": "zhengm58@gmail.com",
+        "discord": "Nooz#0816"
+    },
+    "record": {
+        "CNAME": "noozsbc.github.io/portfolio/"
```

Comment:
Invalid CNAME. CNAME cannot contain slashes.
Check the [docs](https://www.is-a.dev/docs/github_pages/) for more information regarding setting up a is-a.dev subdomain with GitHub pages.

Reply:
Fixed